### ISSUE TITLE
GLM 0.9.8.5 is broken on newer GCC minor versions

### ISSUE THREAD

**Issue by turol** [g-truc/glm#736]
GCC versions 6.4 and 7.3 are not in `glm/simd/platform.h` and the library fails to compile on those compilers. You should cherry-pick the relevant fix from master branch and release 0.9.8.6.

**Comment by scheibel** [comment]
This issue seems related to #728.

**Comment by turol** [comment]
Duplicate of #728

**Comment by galegosimpatico** [comment]
https://github.com/g-truc/glm/blob/6fa203eeb7fbcbb6f620501fad40359c8a456049/glm/simd/platform.h#L278

I think I understand the problem with GCC 7.3 in https://github.com/g-truc/glm/issues/728, but here isn't GCC 6.4 supposed to be matched by that line?

[Whole block](https://github.com/g-truc/glm/blob/6fa203eeb7fbcbb6f620501fad40359c8a456049/glm/simd/platform.h#L246-L290).

**Comment by galegosimpatico** [comment]
I was able to build the use of GLM 0.9.8.4 with GCC 7.3.0 only after fixing [the offending line](https://github.com/g-truc/glm/blob/6fa203eeb7fbcbb6f620501fad40359c8a456049/glm/simd/platform.h#L284) but GCC 6.4.0 did not get offended at the line.
